.github/workflows/publish-spark-metadata-client.yaml

```diff
@@ -39,18 +39,24 @@ jobs:
           cat << EOS > ~/.sbt/sonatype_credentials
           realm=Sonatype Nexus Repository Manager
           host=s01.oss.sonatype.org
-          user=${OSSRH_USERNAME}
-          password=${OSSRH_TOKEN}
+          user=${SONATYPE_USERNAME}
+          password=${SONATYPE_TOKEN}
           EOS
-        env:
+        env:                    # Some credentials required in this step, some in next.
+          AWS_REGION: us-east-1
```

Comment:
can't mark the line but there is something off in line 34 - looks like we have extra redirect.
also, we can just use echo with redirect instead of cat and it will work:
```yaml
run: |
  echo "realm=Sonatype Nexus Repository Manager
  host=s01.oss.sonatype.org
  user=${SONATYPE_USERNAME}
  password=${SONATYPE_TOKEN}" > ~/.sbt/sonatype_credentials
```

Reply:
I will _not_ do this redirect, I believe that "here documents" have simpler quotation rules.  l.34 is definitely broken, so fixing it (I believe that this "realm" is just a string, and the Scala config parser is sufficiently broken that it doesn't care about the extra quotes that got in.)